### PR TITLE
CIV-5788 add CUI filed to case data

### PR DESCRIPTION
**Pull request by kamilewandowski**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-5788

### Change description ###
add respondent LIP response field for case data

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java [hunk 1]
```diff
@@ -5,11 +5,16 @@
...
 
+import uk.gov.hmcts.reform.civil.enums.YesOrNo;
+
 @Data
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
 public class RespondentLiPResponse {
 
+    private YesOrNo partialAdmissionAlreadyPaid;
```
Comment (TatsyMaslova) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java:16]: Not sure whether these should be just as fields as are to do with partial admission. would not it be better to have a model for lop partial admission and have the fields there?

**Review by LeonardoPalmeiro — DISMISSED** [review]

**Review by TatsyMaslova — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.civil.model;
```
Comment (neeta-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java:1]: Merge master into your PR. This changes already exist in master.

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
```
Comment (portizdev) [inline src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java:2]: This file is not in the correct path, you are missing a citizenui folder on the last level. I remember commenting it when you did the demo.

This is the correct path to the file: 

`src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CaseDataLiP.java`

**Review by portizdev — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java [hunk 4]
```diff
@@ -11,6 +12,10 @@
 @AllArgsConstructor
 public class RespondentLiPResponse {
 
+    @JsonUnwrapped
+    private PartAdmitResponseLiP partAdmitResponseLiP;
```
Comment (kenneth-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java:16]: I think this field should have related CCD field, could you please check?

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java (hunk 3 above)
Comment (kamilewandowski) [reply src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java]: I have removed it, the file already exists.

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java (hunk 2 above)
Comment (kamilewandowski) [reply src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java]: resolved

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java (hunk 4 above)
Comment (kenneth-hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java:16]: Related field should be the following - specDefenceAdmittedRequired

Comment (portizdev) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java:16]: Agree with Kenneth - You need to change it also in CUI and CCD prs

Comment (portizdev) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java:16]: partAdmitCUICCD

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by portizdev — DISMISSED** [review]

**Review by PaulH-HMCTS — DISMISSED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by PaulH-HMCTS — APPROVED** [review]

**Pull request merged**